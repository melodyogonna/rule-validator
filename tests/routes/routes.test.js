const supertest = require("supertest");
const app = require("../../src/main");

it("should return relevant personal details", async function () {
  const request = await supertest(app).get("/");

  expect(request.status).toBe(200);
  expect(request.body).toHaveProperty("status", "success");
  expect(request).toMatchSnapshot();
});
