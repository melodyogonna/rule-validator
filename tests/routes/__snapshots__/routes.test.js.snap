// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return relevant personal details 1`] = `
Object {
  "header": Object {
    "connection": "close",
    "content-length": "196",
    "content-type": "application/json; charset=utf-8",
    "date": "Fri, 29 Jan 2021 22:30:28 GMT",
    "etag": "W/\\"c4-obVcZmINjhxgt5yeo3k6rYYAsmc\\"",
    "x-powered-by": "Express",
  },
  "req": Object {
    "data": undefined,
    "headers": Object {},
    "method": "GET",
    "url": "http://127.0.0.1:46359/",
  },
  "status": 200,
  "text": "{\\"message\\":\\"rule validation api\\",\\"status\\":\\"success\\",\\"data\\":{\\"name\\":\\"Anyaegbulam Melody\\",\\"github\\":\\"@melodyogonna\\",\\"email\\":\\"melodyogonna@gmail.com\\",\\"mobile\\":\\"08122522386\\",\\"twitter\\":\\"@melodyogonna\\"}}",
}
`;
